pennylane/templates/embeddings/qaoa.py

```diff
@@ -197,14 +139,12 @@ def circuit(weights, f=None):
 
         **Using parameter initialization functions**
 
-        The initial weight parameters can alternatively be generated by utility functions from the
-        ``pennylane.init`` module, for example using the function :func:`~.qaoa_embedding_normal`:
+        A random numpy weights array can be generated using the static methods
+        `QAOAEmbedding.weights_normal` and `QAOAEmbedding.weights_uniform`.
 
         .. code-block:: python
 
-            from pennylane.init import qaoa_embedding_normal
-            weights = qaoa_embedding_normal(n_layers=2, n_wires=2, mean=0, std=0.2)
-
+            weights = QAOAEmbedding.weights_normal(n_layers=2, n_wires=2, mean=0, std=0.2)
```

Comment:
An old issue was to move the init functions closer to the template, which we can do now. 

At some stage we may want to even initialise automatically, but this is an extremely hard feature to support in all interfaces.